packages/orchestration/src/exos/README.md

```diff
@@ -9,11 +9,11 @@ classDiagram
     LCAKit --* LocalchainAccount
     ICQConnectionKit --* Port
     ICQConnectionKit --* Connection
-    ChainAccountKit --* Port
-    ChainAccountKit --* Connection
+    IcaAccountKit --* Port
```

Comment:
line 3 says,
> As of 2024-05-29…

This change means that's not true anymore, but it's not accurate as of today either. 

Please change the message to something like "Last verified 2024-05-29"

To update it before release I've added an item to https://github.com/Agoric/agoric-sdk/issues/9757

Reply:
I didn't look closely in my find and replace but it's pretty dated. Please see: [535c256](https://github.com/Agoric/agoric-sdk/pull/9818/commits/535c256d605a8f3cd869b5be0fe2ff62a7a5d8cd)